.idea/misc.xml

```diff
@@ -3,7 +3,7 @@
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="ES6" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="11" project-jdk-type="JavaSDK">
```

Comment:
You want to be a little careful about including things like this in a pull request that's arguably about something entirely different. In this case (because there's just the two of you), bumping up from Java 8 to 11 isn't a huge deal, but in general you'd want that to be a thing that is discussed and done as its own change, with documentation on _why_ you'd want to move to 11, etc., etc.

Reply:
That was an accidental change that happened because I only have Java 11, I will change that back.